index.template.html

```diff
@@ -402,6 +413,9 @@ <h3>Report Recovery</h3>
             <div id="zoom_warning" class="lfloat slickbox" style="width:250px; bottom: 0px;   left: 50%; transform: translate(-50%, -50%);">
                 <div style="width:250px">Zoom in for realtime data!</div>
             </div>
+            <div id="recovery_notice" class="lfloat slickbox" onclick="show_recoveries();" style="width:270px; bottom: 0px; left: 50%; transform: translate(-50%, -50%); display:none;">
+                <div style="width:270px"><b>Click here to report a sonde recovery!<b/></div>
```

Comment:
The /b here isn't closed properly, this is probably causing the rendering issues